react/features/base/connection/actions.any.ts

```diff
@@ -194,3 +200,206 @@ export function setLocationURL(locationURL?: URL) {
+                JitsiConnectionEvents.CONNECTION_ESTABLISHED,
+                _onConnectionEstablished);
+            connection.removeEventListener(
+                JitsiConnectionEvents.CONNECTION_FAILED,
+                _onConnectionFailed);
+            dispatch(connectionEstablished(connection, Date.now()));
+        }
+
+        /**
+         * Rejects external promise when connection fails.
+         *
+         * @param {JitsiConnectionErrors} err - Connection error.
+         * @param {string} [msg] - Error message supplied by lib-jitsi-meet.
+         * @param {Object} [credentials] - The invalid credentials that were
+         * used to authenticate and the authentication failed.
+         * @param {string} [credentials.jid] - The XMPP user's ID.
+         * @param {string} [credentials.password] - The XMPP user's password.
+         * @param {Object} details - Additional information about the error.
+         * @private
+         * @returns {void}
+         */
+        function _onConnectionFailed( // eslint-disable-line max-params
+                err: string,
+                msg: string,
+                credentials: any,
+                details: Object) {
+            connection.removeEventListener(
+                JitsiConnectionEvents.CONNECTION_ESTABLISHED,
+                _onConnectionEstablished);
+            connection.removeEventListener(
+                JitsiConnectionEvents.CONNECTION_FAILED,
+                _onConnectionFailed);
+
+            logger.error('CONNECTION FAILED:', err);
+            dispatch(
+                connectionFailed(
+                    connection, {
+                        credentials,
+                        details,
+                        name: err,
+                        message: msg
+                    }
+                ));
+        }
+
+        /**
+         * Unsubscribe the connection instance from
+         * {@code CONNECTION_DISCONNECTED} and {@code CONNECTION_FAILED} events.
+         *
+         * @returns {void}
+         */
+        function unsubscribe() {
+            connection.removeEventListener(
+                JitsiConnectionEvents.CONNECTION_DISCONNECTED,
+                _onConnectionDisconnected);
+            connection.removeEventListener(
+                JitsiConnectionEvents.CONNECTION_FAILED,
+                _onConnectionFailed);
+        }
+    };
+}
+
+/**
+ * Create an action for when a connection will connect.
+ *
+ * @param {JitsiConnection} connection - The {@code JitsiConnection} which will
+ * connect.
+ * @private
+ * @returns {{
+ *     type: CONNECTION_WILL_CONNECT,
+ *     connection: JitsiConnection
+ * }}
+ */
+function _connectionWillConnect(connection: Object) {
+    return {
+        type: CONNECTION_WILL_CONNECT,
+        connection
+    };
+}
+
+/**
+ * Closes connection.
+ *
+ * @returns {Function}
+ */
+export function disconnect() {
+    /* eslint-enable @typescript-eslint/no-unused-vars */
```

Comment:
Why do you need this? If you need to declare a parameter you don't use you can do it like so: `_param` and the linter won't complain.